UI/CUIWnd.cpp

```diff
@@ -318,7 +318,10 @@ void CUIWnd::Render() {
         glDrawArrays(GL_LINE_LOOP,      m_buffer_indices[0].first, m_buffer_indices[0].second);
 
     } else {
-        glColor(ClientUI::WndColor());
+        if (m_flashing && GG::GUI::GetGUI()->Ticks() % (m_flash_duration * 2) > m_flash_duration)
+            glColor(GG::LightColor(ClientUI::WndColor()));
+        else
+            glColor(ClientUI::WndColor());
         glDrawArrays(GL_TRIANGLE_FAN,   m_buffer_indices[1].first, m_buffer_indices[1].second);
         glColor(ClientUI::WndOuterBorderColor());
```

Comment:
Should probably also flash the border outlines

Reply:
Sounds good.